interfaces/acados_template/acados_template/acados_ocp.py

```diff
@@ -2198,6 +2211,7 @@ def __init__(self, acados_path=''):
         self.acados_lib_path = f'{acados_path}/lib'
 
         self.__parameter_values = np.array([])
+        self.__problem_class = 'OCP'
```

Comment:
maybe stupid question, but what is this for?

Reply:
+1 for the stupid question :p

Reply:
I added this in order to be able to render the integrator C code from both an OCP formulation `AcadosOcp` and from a pure integrator formulation `AcadosSim` .
There are a few options that should not be available for the OCP, but it is useful to be able to set them in the integrator formulation.
https://github.com/acados/acados/blob/67e594c92dc1adb8fe45f8b54aacddc98b288b38/interfaces/acados_template/acados_template/c_templates_tera/acados_sim_solver.in.c#L261

Reply:
Do you have another suggestion on how to do that?

Reply:
I see, no better suggestions from my side :p